src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +36,8 @@
   
 
   private final DifferentialDriveTrain differentialDriveTrain = new DifferentialDriveTrain(Constants.leftDrivetrainIDs, Constants.rightDrivetrainIDs, Constants.driveTrainMotorType, Constants.driveMotorProfile);
-  private final ArcadeDrive arcadeDrive = new ArcadeDrive(Constants.driverStick, differentialDriveTrain);
+  private final ReversableArcadeDrive reversableArcadeDrive = new ReversableArcadeDrive(Constants.driverStick, differentialDriveTrain);
+  private final SwitchDriveDirrection switchDriveDirrection = new SwitchDriveDirrection(differentialDriveTrain);
```

Comment:
it would be nice if you could spell direction correctly because this is a major class name.

Reply:
fixed it